### PR TITLE
Enhancing docblock information and adding missing tags

### PR DESCRIPTION
**Pull request by Sjaure**
### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/functions/template-tags/link.php [hunk 1]
```diff
@@ -19,11 +19,13 @@ function tribe_the_prev_event_link( $anchor = false ) {
 }
 
 /**
- * Return a link to the previous event by start date for the given event.
+ * Returns a link to the previous event by start date for the given event.
  *
- * @param bool|string $anchor link text. Use %title% to place the post title in your string.
+ * @since 5.14.0
+ * 
+ * @param bool|string $anchor (optional) The link text. Use %title% to place the post title in your string. Default is false
```
[the remark this answers is not in the document]
Comment (bordoni) [reply src/functions/template-tags/link.php:26]: Make sure this is addressed.

**Review by bordoni — CHANGES_REQUESTED** [review]

**Comment by bordoni** [comment]
Add the `Code Review` label and assign yourself.

**Comment by Sjaure** [comment]
@bordoni , I added the missing tag, assigned it to myself. Also added the missing punctuation and comments on readme.txt
I guess someone else needs to merge the changes to that file or should I go ahead and resolve it?

**Comment by Sjaure** [comment]
@bordoni, I fixed a couple of issues with the changelog file and resolved the conflicts too.

File: readme.txt [hunk 2]
```diff
@@ -234,10 +234,17 @@ Remember to always make a backup of your database and files before updating!
...
+* Tweak - Add note to `tribe_create_organizer()` docblock to indicate future deprecation.
+* Tweak - Add note to `tribe_create_event()` docblock to indicate future deprecation.
+* Tweak - Add note to `tribe_event_update()` docblock to indicate future deprecation.
+* Tweak - Add information and missing tags to `tribe_get_prev_event_link()`.
+=======
 * Feature - Adapt to using the refactored tribe_asset. Remove some unused asset calls. [TCMN-172]
 * Fix - Remove automatic capitalization for 'View Calendar' text on Calendar List Widget. [TECTRIA-40]
 * Tweak - Add note to `tribe_create_venue()`, `tribe_create_organizer()`, `tribe_create_event()`, `tribe_update_event()`, `tribe_update_venue()`, `tribe_update_organizer()`, and `tribe_delete_venue()` docblocks to indicate future deprecation.
 * Tweak - Add docblocks to `src/Tribe/Featured_Events/Permalinks_Helper.php`.
+>>>>>>> release/T24.asag
```
Comment (bordoni) [inline readme.txt:247]: Resolve the conflict.

**Review by bordoni — CHANGES_REQUESTED** [review]

**Comment by bordoni** [comment]
Move to `release/T24.babayaga`

**Pull request closed without merging**